lib/control.mli

```diff
@@ -8,6 +8,9 @@
 
 (** Global control of Coq. *)
 
+(** Will periodically call [Thread.delay] if set to true *)
+val enable_yield : bool ref
```

Comment:
Yield or delay? I'd say "force_thread_fairness"

Reply:
Technically it is a yield, isn't it?

Reply:
I'd love to do a yield, but since it does not work I do a delay.

This technical detail has nothing to do with the boolean flag.
Since `Threads.yield` does a `yield`, while `Threads.delay` does a `delay`, the boolean flag should be called `enable_delay`.